hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AbfsConfiguration.java

```diff
@@ -411,6 +419,10 @@ public String getAzureAtomicRenameDirs() {
     return this.azureAtomicDirs;
   }
 
+  public String getAppendBlobDirs() {
```

Comment:
Why not return a HashSet as all callers are going to need it so.

Reply:
this is just the way it is done for other config parameter for dirs. see: getAzureAtomicRenameDirs